core/src/main/resources/error/error-classes.json

```diff
@@ -851,6 +851,11 @@
       "Cannot name the managed table as <identifier>, as its associated location <location> already exists. Please pick a different table name, or remove the existing location first."
     ]
   },
+  "MALFORMED_CSV_RECORD" : {
+    "message" : [
+      "Malformed CSV record"
```

Comment:
Can we provide more info to users? At least which record is malformed, and where is the problem - schema mismatch, the converter failed, and so on.

Reply:
Thanks!
I think we can provide the schema information, since it raised when schema mismatch.
Just add some more informations to error message.